### PR TITLE
Added a tooltip to sunburst.  

### PR DESCRIPTION
**Pull request by liquidpele**
I tried to re-use the existing tooltip code, but it's way too specific for x-y coordinates and having a mouse following version is stupid simple anyway since it doesn't need any of the extra complexity.

### CODE REVIEW

File: src/models/sunburstChart.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        , noData = null
+        , duration = 250
+        , dispatch = d3.dispatch('tooltipShow', 'tooltipHide', 'stateChange', 'changeState','renderEnd')
+        ;
+
+    //============================================================
+    // Private Variables
+    //------------------------------------------------------------
+
+    var tooltip = d3.select('body')
```
Comment (robinfhu) [inline src/models/sunburstChart.js]: Honestly, I really like this way of doing tooltips.

**Comment by robinfhu** [comment]
No real objections, keep up the awesome work!

**Comment by liquidpele** [comment]
Cool, merging in.

**Comment by liquidpele** [comment]
@robinfhu what do you like about that tooltip code?  I'm planning to convert the pie chart to similar so I was just curious :)

**Comment by robinfhu** [comment]
First, I've always supported the idea of putting the div on the body, because it is much easier to absolutely position it.  All you need to do then is position the top and left using:

hoverElement.getBoundingClientRect().top + window.pageYOffset
hoverElement.getBoundingClientRect().left

**Pull request merged**